redis.conf

```diff
@@ -1382,6 +1382,10 @@ disable-thp yes
 # If the AOF is enabled on startup Redis will load the AOF, that is the file
 # with the better durability guarantees.
 #
+# Note that changing this value in a config file of an existing database and
+# restarting the server can lead to data loss. A conversion needs to be done
+# by setting it via CONFIG command on a live server first.
```

Comment:
Do we need to mention that the reason we do this is because we need to trigger an AOFRW?

or mention doing a CONFIG SET appendonly yes will trigger an AOFRW

Reply:
we can, if we can make it short.
alternatively, maybe the context above that line is enough and the users has a link for further reading just below it...

Reply:
Yeah, in my opinion the "Please check https://redis.io/topics/persistence for more information." right below should be sufficient for people that want to know more.